domain_content/domain_content.module

```diff
@@ -33,3 +33,37 @@ function domain_content_domain_operations(DomainInterface $domain, AccountInterf
 
   return $operations;
 }
+
+/**
+ * Implements hook_requirements.
+ */
+function domain_content_requirements($phase) {
+  $requirements = array();
```

Comment:
Nitpick: use array short-form. Especially that core has moved to that - https://groups.drupal.org/node/516354